ubuntu-github-actions--20.04/Dockerfile

```diff
@@ -1,14 +1,17 @@
-# Image name: quay.io/opencv-ci/opencv-ubuntu:20.04
+# Version: 4.6.0-1.0.0
+# Image name: quay.io/opencv-ci/opencv-ubuntu-20.04:4.6.0-1.0.0
```

Comment:
I propose new image name and tags. The tag will have several parts:
- os and platform - `opencv-ubuntu-20.04` / `opencv-ubuntu-20.04-arm64`
- current OpenCV version - `4.6.0`
- dockerfile version - `1.0.0`

Reply:
> current OpenCV version - 4.6.0

Not sure if we really need this.
In general, we could share the same docker image between different OpenCV branches.

> dockerfile version - 1.0.0

This doesn't make a lot of sense (as we can't describe difference between 1.0.0 and 2.0.0 or that we expect in 3.0.0).
Timestamp like `20220621` should provide less confusion.

Reply:
Changed the previous logic to timestamp.